src/settings/util/connectionProblem.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import {
+  get,
+} from 'lodash';
+
+import {
+  CALLOUT_ERROR_TYPE,
+  ERROR_TYPE_NOT_UNIQUE,
+} from '../constants';
+
+// eslint-disable-next-line import/prefer-default-export
+export const connectionProblem = async (response, showCallout) => {
+  const errors = await response.json();
+  const errorType = get(errors, ['errors', 0, 'type'], '');
+
+  if (errorType === ERROR_TYPE_NOT_UNIQUE) {
+    showCallout({
+      type: CALLOUT_ERROR_TYPE,
+      message: <FormattedMessage id="ui-oai-pmh.settings.sets.callout.validationError.notUnique" />,
+    });
+  } else {
```

Comment:
Looks like all other cases (other than notUnique message) will return message about a connection problem '`module error` -- I see this message if specification is empty as well

Is there a reason why messages from the server side are not used?

https://github.com/folio-org/mod-oai-pmh/blob/2daab338c0022e8da72f9c8792c36c20a32d2a60/src/main/java/org/folio/rest/impl/OaiPmhSetImpl.java#L158


<img width="1678" alt="Screen Shot 2020-09-08 at 11 42 33 AM" src="https://user-images.githubusercontent.com/19415226/92498805-10280480-f1c9-11ea-8408-be3b47c2f163.png">

Reply:
In scope of the current task, we add error handling for non-unique values (This is integration with https://issues.folio.org/browse/MODOAIPMH-220). UI validation for other cases will be implemented in scope of https://issues.folio.org/browse/UIOAIPMH-12.

Reply:
So long as other cases will be handled -- I think the module message could be confusing otherwise.  

The name of the function `connectionProblem` was a little misleading since it will handle different types of errors from back end (including validation errors)